src/client/components/Inputs/Select/hooks/useComponents.ts

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from 'react'
+import { Props as ReactSelectProps } from 'react-select'
+
+import {
+  ClearIndicator,
+  DropdownIndicator,
+  IndicatorsContainer,
+  MultiSelectOption,
+} from 'client/components/Inputs/Select/Indicators'
+import { SelectProps } from 'client/components/Inputs/Select/types'
+
+type Returned = ReactSelectProps['components']
+
+export const useComponents = (props: SelectProps): Returned => {
+  const { isMulti } = props
+
+  return useMemo<Returned>(() => {
+    return {
+      ClearIndicator,
+      DropdownIndicator,
+      IndicatorsContainer,
+      IndicatorSeparator: null,
+      Option: isMulti ? MultiSelectOption : null,
```

Comment:
This is breaking the single select. If isMulti is false, then Option becomes null. React-select then uses null when rendering Options, instead of using the default component. This would also happen if you do Option: undefined. 
Instead you need to set it with something like:
```
    if (shouldUseCustomOption) {
      components.Option = optionComponent
    }

```

Reply:
Fix: https://github.com/openforis/fra-platform/pull/3752